server/models/index.ts

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import fs from 'fs';
+import path from 'path';
+import Sequelize from 'sequelize';
+const config = require( '../config/config');
+const basename = path.basename(__filename);
+const db: any = {};
+
+const sequelize = new Sequelize.Sequelize(config.DB_URL,  {
+  dialect: config.dialect,
+  dialectOptions: {
+    ssl:  { /* Required for our host */
+      rejectUnauthorized: false,
+    }
+  }
+});
+
+fs
+  .readdirSync(__dirname)
+  .filter((file: string) => {
+    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
+  })
+  .forEach((file: any) => {
+    const model = require(path.join(__dirname, file))(sequelize, Sequelize.DataTypes);
+    db[model.name] = model;
+  });
+
+Object.keys(db).forEach(modelName => {
+  if (db[modelName].associate) {
+    db[modelName].associate(db);
+  }
+});
+
+db.sequelize = sequelize;
+db.Sequelize = Sequelize;
+
+export default db;
```

Comment:
Is there a way to autogenerate some code? 

See https://github.com/sequelize/sequelize-auto or similar tools